arch/x86/core/intel64/cpu.c

```diff
@@ -7,6 +7,9 @@
 #include <kernel_arch_data.h>
 #include <kernel_arch_func.h>
 #include <kernel_structs.h>
+#include <arch/x86/multiboot.h>
+
+extern FUNC_NORETURN void z_x86_prep_c(struct multiboot_info *);
```

Comment:
probably better to just put this prototype in kernel_arch_func.h.
Not sure if our coding standard allows prototypes without argument names.

Reply:
I've pushed a commit to move the prototype. Fixed.